USAGE.md

```diff
@@ -0,0 +1,386 @@
+    - [Split Tunneling](#split-tunneling)
+  - [Enhancements](#enhancements)
+    - [Disable sudo password query](#disable-sudo-password-query)
+    - [Configure alias for quicker access](#configure-alias-for-quicker-access)
+    - [Auto-connect on boot](#auto-connect-on-boot)
+      - [via Systemd Service](#via-systemd-service)
+
+## Installation & Updating
+
+### Installing Dependencies
+
+**Dependencies:**
+
+- openvpn
+- dialog (optional, needed for interactive selection)
+- pip for python3 (pip3)
+- python3.5+
+
+Run the following command to install the dependencies on your distribution
+
+| **Distro**                              | **Command**                                     |
+|:----------------------------------------|:------------------------------------------------|
+|Fedora/CentOS/RHEL                       | `sudo dnf install -y openvpn dialog python3-pip`|
+|Ubuntu/Linux Mint/Debian and derivatives | `sudo apt install -y openvpn dialog python3-pip`|
+|OpenSUSE/SLES                            | `sudo zypper in -y openvpn dialog python3-pip`  |
+|Arch Linux/Manjaro                       | `sudo pacman -S openvpn dialog python-pip`      |
+
+### Installing ProtonVPN-CLI
+
+Installation happens via Python's package manager PIP.
+
+*Note: Make sure to run pip with sudo, so it installs globally and recognizes the command with sudo*
+
+`sudo pip3 install protonvpn-cli`
+
+### Updating ProtonVPN-CLI
+
+`sudo pip3 install protonvpn-cli --upgrade`
+
+### Initialization
+
+Before being able to use ProtonVPN-CLI you need to initialize it. For this run `sudo protonvpn init` and follow the prompts on the screen.
+
+### Uninstallation
+
+If you want to uninstall ProtonVPN-CLI, run `configure` first and purge the configuration. Then uninstall through the package manager you used for installation.
+
+For PIP this would be
+
+`sudo pip3 uninstall protonvpn-cli`
+
+Bye Bye 😔
+
+### Example Installation on Ubuntu 18.04
+
+1. Installing dependencies
+
+    To install ProtonVPN-CLI's dependencies, open a terminal and type `sudo apt install -y dialog openvpn python3-pip` and confirm with Enter. Wait for the installations to finish
+
+2. Installing ProtonVPN-CLI
+
+    To install ProtonVPN-CLI type `sudo pip3 install protonvpn-cli` in the terminal and confirm with Enter again. This should look something like this:
+
+    ![ubuntu-pip-install](https://i.imgur.com/jSuftoe.png)
+
+3. Initialize the ProtonVPN profile
+
+    Now you have access to the `protonvpn` command. Before using ProtonVPN-CLI you need to initialize your profile. For this type `sudo protonvpn init`
+
+    This will first ask you for your OpenVPN username and password. You can find them on https://account.protonvpn.com/account
+
+    ![openvpn-pass](https://i.imgur.com/EdZ01T9.png)
+
+    Enter these details in the prompts
+
+    ![ubuntu-pass-entry](https://i.imgur.com/Vrcq2oO.png)
+
+    Next you need to select your plan. If you are a trial user currently, select `3) Plus`.
```

Comment:
This line could be made more clear by writing "If you are currently enjoying the 7-day free trial of ProtonVPN Plus, select `3) Plus`
```suggestion
    Next you need to select your plan. If you are currently enjoying the 7-day free trial of ProtonVPN Plus, select `3) Plus`.
```